addon/ng2/tasks/install.ts

```diff
@@ -0,0 +1,128 @@
+import * as Task from 'ember-cli/lib/models/task';
+import * as npmTask from 'ember-cli/lib/tasks/npm-task';
+import * as chalk from 'chalk';
+import * as path from 'path';
+import { EventEmitter } from 'events';
+import * as search from 'typings-core/dist/search';
+import * as typings from 'typings-core/dist/install';
+import * as systemJS from '../utilities/systemjs-helper.ts';
+
+module.exports = Task.extend({
+  completionOKMessage: 'Successfully installed.',
+  completionErrorMessage: 'Error installing package.',
+
+  run: function(options) {
+    this.packages = options.packages;
```

Comment:
It gets `object` with data from `install` command, I just name it `options` in case we'll add something later, currently is just the packages it gets from arguments.